### PR TITLE
Vuetify should work now

### PR DESCRIPTION
**Pull request by te4-david-petersson**
Vuetify added

### CODE REVIEW

File: frontend/public/index.html [hunk 1]
```diff
@@ -8,6 +8,10 @@
     <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
     <script src="https://apis.google.com/js/client:platform.js"></script>
     <title>yabs</title>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:100,300,400,500,700,900">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@mdi/font@latest/css/materialdesignicons.min.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:100,300,400,500,700,900">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@mdi/font@latest/css/materialdesignicons.min.css">
```
Comment (TE4-jacob-lundberg) [inline frontend/public/index.html]: Duplication really needed? Might be autogenerated but could maybe be removed?

File: frontend/src/App.vue [hunk 2]
```diff
@@ -1,60 +1,56 @@
-<template lang="pug">
```
Comment (TE4-jacob-lundberg) [inline frontend/src/App.vue]: You overwrote the origin content of this file, should probably change it back so the site still works

File: frontend/src/components/HelloWorld.vue [hunk 3]
```diff
@@ -0,0 +1,146 @@
+<template>
```
Comment (TE4-jacob-lundberg) [inline frontend/src/components/HelloWorld.vue]: This is autogenerated, you should be able to remove this file

File: frontend/src/views/Home.vue [hunk 4]
```diff
@@ -0,0 +1,16 @@
+<template>
```
Comment (TE4-jacob-lundberg) [inline frontend/src/views/Home.vue]: Should also be able to remove this file

File: frontend/public/index.html (hunk 1 above)
Comment (te4-david-petersson) [reply frontend/public/index.html]: check the other pull request

**Pull request closed without merging**